react/.storybook/preview.js

```diff
@@ -1,58 +1,92 @@
+import prettier from 'prettier/standalone';
+import parserHtml from 'prettier/parser-html';
+import { Prism as SyntaxHighlighter } from 'react-syntax-highlighter';
+import { okaidia } from 'react-syntax-highlighter/dist/esm/styles/prism';
 import '../src/index.scss';
 
 const storyKindOrder = [
   'about', // storyKindOrder.indexOf -1 follow alphabetical order
   'brand', // storyKindOrder.indexOf -1 follow alphabetical order
   'dataviz', // storyKindOrder.indexOf -1 follow alphabetical order
-  'forms|atoms',
-  'forms|molecules',
-  'forms|organisms',
-  'forms|context',
+  'forms',
   'atoms',
   'molecules',
   'organisms',
-  'templates',
-  'pages'
+  'others/templates',
+  'others/pages'
 ];
 
-addParameters({
-  options: {
-    storySort: (a, b) => {
-      const aKind = a[1].kind.split('/')[0];
-      const bKind = b[1].kind.split('/')[0];
-      return (storyKindOrder.indexOf(aKind) - storyKindOrder.indexOf(bKind)) || a[1].id.toLowerCase().localeCompare(b[1].id.toLowerCase(), undefined, { numeric: true });
-    }
-  }
-});
-
-addDecorator(
-  withInfo({
-    styles: {
-      button: {
-        base: {
-          fontFamily: 'sans-serif',
-          fontSize: '14px',
-          fontWeight: '500',
-          display: 'block',
-          position: 'fixed',
-          border: 'none',
-          background: '#14558f',
-          color: '#fff',
-          padding: '5px 15px',
-          cursor: 'pointer',
-        },
-        topRight: {
-          bottom: 0,
-          right: 0,
-          top: 'unset',
-          borderRadius: '5px 0 0 0',
-        }
+export const StoryPage = ({ StoryComponent = null, showStories = false, Description }) => {
+  const docsContext = React.useContext(DocsContext);
+  const [showHTML, setShowHTML] = React.useState(false);
+  const { id, name, parameters = {}, args } = docsContext;
+  const { component } = parameters;
+  const HtmlComponent = StoryComponent || component;
+  let html = null;
+  if (HtmlComponent) {
+    html = prettier.format(ReactDOMServer
+    .renderToStaticMarkup(
+      (
+        <HtmlComponent {...args} />
+      )),
+      {
+        htmlWhitespaceSensitivity: 'ignore',
+        endOfLine: 'auto',
+        parser: 'html',
+        plugins: [parserHtml]
       }
-    }
-  })
-);
+    );
+  }
+  const actionItem = {
+    title: showHTML ? 'Hide HTML' : 'Show HTML?', 
+    onClick: () => setShowHTML((prev) => !prev)
+  };
+  return(
+    <>
+      <Title>Description</Title>
+      <Subtitle />
+      { Description && <Description />}
+      <Primary name={name} />
+      {html && (
+        <Heading>
+          HTML
+          <ActionBar actionItems={[actionItem]} />
+        </Heading>
+      )}
+      {!showHTML && <Source storyId={id} error="Click Show HTML above to view markup source." />}
+      {html && showHTML && <Source storyId={id} language="html" code={html} dark />}
+      <ArgsTable story={CURRENT_SELECTION}/>
+      { showStories && <Stories />}
```

Comment:
I'm not seeing stories being shown in docs tab

Reply:
I made it hidden by default in the docs tab, but I can make it shown by default.

Reply:
hmm I'm not sure how useful it is to render stories now that there's React code snippets already rendered in docs below the live component. 
It feels repetitive for the docs to render components twice and with the React code 
e.g. http://localhost:6006/?path=/docs/organisms-calloutalert--callout-alert-real-data

Reply:
@smurrayatwork ^